kratos/processes/find_nodal_neighbours_process.h

```diff
@@ -254,10 +195,9 @@ class FindNodalNeighboursProcess
     ///@name Member Variables
     ///@{
     ModelPart& mr_model_part;
-    unsigned int mavg_elems;
-    unsigned int mavg_nodes;
-
 
+    std::unique_ptr<FindGlobalNodalElementalNeighboursProcess> mpElemNeighboursCalculator = nullptr;
+    std::unique_ptr<FindGlobalNodalNeighboursProcess> mpNodeNeighboursCalculator = nullptr;
```

Comment:
why pointers?

Reply:
well, the previous implementation was allowing for either (nodes or elements) to be null

we now removed this option (see conversation with Pooyan) but i did minimal changes in the code (simply taking out the ifs) so the pointers remained.

Considering that this has essentially no overhead and that the overall process should be removed i thought it was not worth to make bigger changes